index.js

```diff
@@ -1,27 +1,27 @@
+/*
+This is a proposal on how to achieve a safety net for modules that are removed from the app.
+If that is the case we want to know at start time that something is not correct.
 
-// // production code
-// const doMagic = require('./command/do-magic').doMagic
-// doMagic({color: 'magenta', message: 'You are one nice cow!'})
-//
-//
-//
-// // test code
-// const createDoMagicCommand = require('./command/do-magic').createDoMagicCommand
-//
-// const testDoMagic = createDoMagicCommand({
-//   chalk: require('chalk'),
-//   cowsay: {
-//     say({text}) {
-//       return `🐮  ${text}`
-//     }
-//   },
-//   log: console.log
-// })
-//
-// testDoMagic({color: 'blue', message: 'Test cows everywhere!'})
+It's easly achiavable with direct require of those modules and avoiding index.js:
+ */
 
 
-// throws an erro
-const doIndexMagic = require('./command')
+// production code
+const doMagic = require('./command/do-magic').doMagic
+doMagic({color: 'magenta', message: 'You are one nice cow!'})
```

Comment:
@QuotableWater7 yeah agree, I just think it's too much boilerplate :| When ES6 modules come in (1 year or so + few month to upgrade) that would be obsolete since `import { foo, bar } from 'exports-only-bar' would fail in runtime, I think. 

My feel that hard requirement to have an integration tests would cover all of this.